src/ImageViewer/ImageViewer.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+import styles from './ImageViewer.module.scss';
+import { images } from './imageslist'
+
+
+interface GridItem {
+    id: number;
+    sol: number;
+    cameraName: string;
+    cameraFullName: string;
+    imageUrl: string;
+    earthDate: string;
+    roverName: string;
+}
+
+export const ImageViewer: FunctionComponent = () => {
+
+
+    const [gridItems, setGridItems] = useState<GridItem[]>(images)
+
+
+    useEffect(() => {
+        setGridItems(images)
+    }, []);
```

Comment:
this doesn't really do anything...
is currently the same as just using `images`.

Is there a reason to add the state and the useEffect?